Minio/ApiEndpoints/IObjectOperations.cs

```diff
@@ -30,9 +31,8 @@ public interface IObjectOperations
         /// </summary>
         /// <param name="bucketName">Bucket to retrieve object from</param>
         /// <param name="objectName">Name of object to retrieve</param>
-        /// <param name="callback">A stream will be passed to the callback</param>
         /// <param name="cancellationToken">Optional cancellation token to cancel the operation</param>
-        Task GetObjectAsync(string bucketName, string objectName, Action<Stream> callback, CancellationToken cancellationToken = default(CancellationToken));
+        Task<byte[]> GetObjectAsync(string bucketName, string objectName, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
this will change the Api method signature - isn't a callback stream as easy to process as a byte array?